modules/nw-dns-operator-logs.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-operator-logs-{context}"]
```

Comment:
s/logs-{context}/logs_{context}

Reply:
This change came up over the weekend, switching the last `-` in the context statement to `_`.